sw/device/silicon_creator/mask_rom/mask_rom_epmp.h

```diff
@@ -43,8 +44,9 @@ extern "C" {
  * to the hardware configuration.
  *
  * @param[out] state The shadow registers to configure with initial values.
+ * @param[in] lc_state The current lifecycle state to check for debug enable.
```

Comment:
nit: you can just use @param for input parameters for consistency.

Reply:
Updated